quickstart.sh

```diff
@@ -7,7 +7,7 @@ echo
 echo " This script will help you to quickly start a TestRail instance and will"
 echo " populate a '.env' file with the necessary configuration values."
 echo " For more advanced configuration please directly modify the .env file and utilize"
-echo " docker-compose directly."
+echo " docker-compose/docker compose directly."
```

Comment:
@acuanico-tr-galt These doesn't seem related to chrome-php updates. Is this for your local only?

Reply:
@parboleda-tr-galt I included these updates to support newer version of docker compose (Compose v2).

Reply:
Let's separate this docker-compose changes in another PR